x-pack/test/functional/apps/lens/group6/legacy_metric.ts

```diff
@@ -34,7 +34,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       });
     });
 
-    it('should color the metric text based on value', async () => {
+    it.skip('should color the metric text based on value', async () => {
```

Comment:
waiting for this to be resolved, before merging

Reply:
as Cee wrote this should be solved with the change to `style` in the legacy metric. The test was checking the style attribute.
Even better, if you want to spend a couple of minute more, you can probably fix the test by checking the computed style instead of the style attribute. the `getLegacyMetricStyle` can just return the `background-color` and `color` from the computes style so we don't depend anymore from style/class